amundsen_application/api/search/v0.py

```diff
@@ -107,35 +113,19 @@ def _create_url_with_field(*, search_term: str, page_index: int) -> str:
 @action_logging
 def _search_user(*, search_term: str, page_index: int) -> Dict[str, Any]:
     """
-        call the search service endpoint and return matching results
-        :return: a json output containing search results array as 'results'
+    call the search service endpoint and return matching results
+    :return: a json output containing search results array as 'results'
 
-        Schema Defined Here: https://github.com/lyft/
-        amundsensearchlibrary/blob/master/search_service/api/search.py
+    Schema Defined Here: https://github.com/lyft/
+    amundsensearchlibrary/blob/master/search_service/api/search.py
```

Comment:
the file should be https://github.com/lyft/amundsensearchlibrary/blob/master/search_service/api/user.py